### ISSUE TITLE
tooltips still appearing off screen on 2.90

### ISSUE THREAD

**Issue by pg1671** [usablica/intro.js#806]
## Description
Tooltips that are too close to the top of the screen still appear with the top off screen with the latest code after positioning fixes. This could be due to use a custom theme with much larger tooltips .. but there is still plenty of room on screen.  I even tried to manually set the position to bottom but this did not affect it.

## Environment
Testing in Chrome and Firefox.

## Actual result
Screenshot included .. top of screenshot is the top of the screen with about 100px cut off .. more than enough room below.
<img width="838" alt="introjs_offscreen" src="https://user-images.githubusercontent.com/616457/34500640-5a481760-f057-11e7-9440-921606887662.png">

## Expected behavior
Tooltip should appear below the highlighted item (or at least over it but with the top on screen).

**Comment by pg1671** [comment]
You can see it here .. https://www.mypresences.com/services/ ... click the ? icon if it does not appear automatically.

**Comment by pg1671** [comment]
Hi .. the problem is no longer visible on the site as I have now added some CSS to stop the box from appear off screen in this case.  You can still use the screenshot as reference.

**Comment by loonix** [comment]
Hi, I am having the same issue... does anyone know any workaround for this?

**Comment by TolstoyDotCom** [comment]
I had the same problem [here](https://tolstoy.com/twittercensorship/report_hassanrouhani_2019_11_12_05_19_54_li.html)

Specifically, the tooltip for the 'Hidden' column (after the 'Suppressed' column) had about half of it above the top of the screen. I tried various options and none of them seemed to do anything. Then I looked at the pg1671 page and it appears he widened the tooltip. I added this after including the intro CSS and it appears to have resolved it at least on desktops (a couple are still partly offscreen on mobile):
`		.introjs-tooltip {
			padding:1.5rem;
			min-width:30rem;
			max-width:40rem;
		}
`
